resource-definitions/k8s-cluster-git/runtime/README.md

```diff
@@ -0,0 +1,11 @@
+## Connecting to a Git repository (GitOps mode)
+
+This section contains example Resource Definitions for connecting to a Git repository to push application CRs ([GitOps mode](https://developer.humanitec.com/integration-and-extensions/humanitec-operator/architecture/#modes-of-operation-humanitec-operator-with-gitops)) and fetch Runtime Information from a different cluster resource:
+
+* [github-for-gitops.yaml](github-for-gitops.yaml): contains configuration for connecting to a Git repository. This Resource Definition co-provisions a [GKE cluster](https://developer.humanitec.com/platform-orchestrator/examples/resource-definitions/#k8s-cluster-gke) to be used to fetch Runtime Information. This format is for use with the [Humanitec CLI](<https://developer.humanitec.com/>).
+* [github-for-gitops.tf](github-for-gitops.tf): contains configuration for connecting to a Git repository. This Resource Definition co-provisions a [GKE cluster](https://developer.humanitec.com/platform-orchestrator/examples/resource-definitions/#k8s-cluster-gke) to be used to fetch Runtime Information. This format is for use with the [Humanitec Terraform provider](<https://registry.terraform.io/>)
+* [gke-temporary-credentials-runtime.yaml](gke-temporary-credentials-runtime.yaml): uses temporary credentials defined via a Cloud Account. The Resource Id specified in the Matching Criteria ensures that this Definition will be matched to provision the Resource co-provisioned by the GitOps cluster Resource Definition. This format is for use with the [Humanitec CLI](https://developer.humanitec.com/platform-orchestrator/cli/)
+* [gke-temporary-credentials-runtime.tf](gke-temporary-credentials-runtime.tf): uses temporary credentials defined via a Cloud Account. The Resource Id specified in the Mathing Criteria ensures that this Definition will be matched to provision by the GitOps cluster Resource Definition. This format is for use with the [Humanitec Terraform provider](https://registry.terraform.io/providers/humanitec/humanitec)
+* [`custom-namespace.yaml`](./custom-namespace.yaml): Shows how to use the Echo Driver to return the name of an externally managed namespace that should match the namespace where the GitOps Operator creates the resources. This format is for use with the [Humanitec CLI](https://developer.humanitec.com/platform-orchestrator/cli/).
+* [`custom-namespace.tf`](./custom-namespace.tf): Shows how to use the Echo Driver to return the name of an externally managed namespace that should match the namespace where the GitOps Operator creates the resources
```

Comment:
It's a _must_ match for the Platform Orchestrator to be able to retrieve runtime data, correct?

```suggestion
* [`custom-namespace.tf`](./custom-namespace.tf): Shows how to use the Echo Driver to return the name of an externally managed namespace that must match the namespace where the GitOps Operator creates the resources
```